app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The ERD diagram shows `author_id` instead of `user_id`. We can solve this issue at the model level to explicitly change the foreign_key column name. We can do this: `belongs_to :author, class_name: 'User'`

Reply:
@elisha2kyakpo1 @hiromataba  by the way I'm not satisfied with the first review that didn't catch all these things and that's cost me a lot of time-wasting